scripts/contractInteraction/testnet_contracts.json

```diff
@@ -23,6 +23,11 @@
     "BPro": "0x4dA7997A819bb46B6758b9102234c289Dd2ad3bf",
     "RSKOracle": "0xE00243Bc6912BF148302e8478996c98c22fE8739",
     "PriceFeedRSKOracle":"0xF2B7440C89431DF82EC6c8F3D079059847565dF0",
+
+    "cSOV1": "0x75bbf7f4d77777730eE35b94881B898113a93124",
```

Comment:
Based on the new commit b78b219, better to change it here as well. And the next line.